02Arrays/Array/Tester.swift

```diff
@@ -73,7 +171,15 @@ class Tester {
     
     
     func maxProfit(array: [Int]) -> Int {
-        1
+        var maxProf = 0
+        var buy = array.min()
```

Comment:
this works assuming the min() is the best day to buy
If the min() was the last element in the array, this would not work